src/Projection/PdoEventStoreProjector.php

```diff
@@ -38,6 +38,10 @@
 
 final class PdoEventStoreProjector implements Projector
 {
+    public const OPTION_PCNTL_DISPATCH = 'trigger_pcntl_dispatch';
```

Comment:
I think the consts should go here: https://github.com/prooph/event-store/blob/master/src/Projection/Projector.php#L25